src/reducers/scrollStateHelper.js

```diff
@@ -299,6 +301,8 @@ export function scrollTo(state, scrollPosition) {
       firstRowIndex: 0,
       firstRowOffset: 0,
       scrollY: scrollY,
+      // TODO (jordan) This overrides the scrollContentHeight changes made by _updateRowHeight
```

Comment:
This erases the work of updateRowHeight in updating scrollContentHeight.  I'm going to remove it unless there's a good reason anyone can think of.